tutorials/composition/Interpolated Waveforms.ipynb

```diff
@@ -0,0 +1,314 @@
+{
```

Comment:
<p>This notebook is very clear ! Thanks for your work ! </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/224/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='19'/>

Reply:
<p>Thanks!</p>